analysis/db/somya/README.md

```diff
@@ -0,0 +1,4 @@
+I was thinking about looking into risk factors for COVID-19. Maybe which ones are most talked about or looked into?
+Something simple to start with might be looking at just the abstracts of a few files and seeing what risk factors they mention and then keeping track of the count.
+I would like to use the risk factors that they identified on [Kaggle](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks?taskId=558).
+Specifically, I would like to start with smoking, co-infections, and pregnancy. 
```

Comment:
Good thinking with risk factors! I believe diabetes, heart disease, etc are ones to also think about. Can you identify all papers in the data set with risk factors mentioned in them, pull what they are, and then determine associations from that? Also, are there any additional datasets published on risk factors?